### ISSUE TITLE
Did you manage to make the cluster mode work with Prisma?

### ISSUE THREAD

**Issue by geosigno** [entrptaher/prisma-issues#1]
Hi, 

I'm following up with [https://github.com/prisma/prisma-client-js/issues/632](https://github.com/prisma/prisma-client-js/issues/632). 

Did you manage to make the cluster mode work with Prisma 2? I'm currently trying to make it work with pm2,  and I don't know where to start actually. Did the code on this repo works?

Thanks!

**Comment by nickreynke** [comment]
@geosigno Did you get it working or is this still an issue? Would like to know whether this works.

//cc @voxelcoder

**Comment by geosigno** [comment]
Yes, at least I'm assuming this is working with the cluster mode of pm2. I didn't make any change, just native pm2 cluster mode.
